src/vulkan_state.cpp

```diff
@@ -62,6 +62,17 @@ static void fillQueueInfo(VkDeviceQueueCreateInfo &info, uint32_t idx,
     info.pQueuePriorities = priorities.data();
 }
 
+VkFormatProperties2 getPhysicalDeviceProperties(const InstanceState &inst,
```

Comment:
change name to something like getFormatProperties. getPhysicalDeviceProperties is an actual vulkan function in the dispatch table that gets more general device properties.